packages/gestalt/src/Callout.js

```diff
@@ -49,43 +52,13 @@ type Props = {|
    * If no `href` is supplied, the action will be a button.
    * The `accessibilityLabel` should follow the [Accessibility guidelines](https://gestalt.pinterest.systems/web/callout#Accessibility).
    */
-  primaryAction?: {|
-    accessibilityLabel: string,
-    disabled?: boolean,
-    href?: string,
-    label: string,
-    onClick?: ({|
-      event:
-        | SyntheticMouseEvent<HTMLButtonElement>
-        | SyntheticMouseEvent<HTMLAnchorElement>
-        | SyntheticKeyboardEvent<HTMLAnchorElement>
-        | SyntheticKeyboardEvent<HTMLButtonElement>,
-      dangerouslyDisableOnNavigation: () => void,
-    |}) => void,
-    rel?: 'none' | 'nofollow',
-    target?: null | 'self' | 'blank',
-  |},
+  primaryAction?: ActionDataType,
```

Comment:
LEt's undo this, Look at the Docs and you'll see this is not captured in the Autogeenrated table docs

Reply:
Not undone, but reworked in a way to make it show in the docs (the type change itself is necessary to fork the component type)